File: String/removedup.py
#User function Template for python3
class Solution:
	def removeDups(self, S):
		# code here
	   
        s=list (S)
        res=""+s[0]
        
	    for i in range (1,len(S)):
	        if not S[i:i+1] in res:
	            res=res+(s[i])
	   
	    return res
	    

#{ 
#  Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
	T=int(input())
	for i in range(T):
		s = input()
		
		ob = Solution()	
		answer = ob.removeDups(s)
		
		print(answer)


# } Driver Code Ends

File: String/removedup.java
// { Driver Code Starts
//Initial Template for Java

import java.io.*;
import java.util.*;

class GFG {
    public static void main(String args[]) throws IOException {
        BufferedReader read =
            new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(read.readLine());
        while (t-- > 0) {
            String s = read.readLine();
            
            Solution ob = new Solution();
            String result = ob.removeDups(s);
            
            System.out.println(result);
        }
    }
}// } Driver Code Ends


//User function Template for Java

class Solution {
    String removeDups(String S) {
        String res=""+S.charAt(0);
        for(int i=1;i<S.length();i++)
        {
            if(!res.contains(S.substring(i,i+1)))
            res=res+S.charAt(i);
            
        }
        return res;
    }
}
